LMS/Billings/Resources/views/card.blade.php

```diff
@@ -0,0 +1,163 @@
+@extends('lms.templates.dashboard')
+@section('breadcrumb')
+    <ol class="navbar-brand breadcrumb mt-4">
+        <li class="breadcrumb-item"><a href="{{ route('dashboard') }}">Cursos</a></li>
+        <li class="breadcrumb-item">Payments</li>
```

Comment:
Verificar se vai ficar em inglês ou em português esses textos aqui!

Acredito também que seria massa no final do projeto realizarmos uma força tarefa pra traduzir esses arquivos, não sei

ps: vou deixar apenas um comentário mas é valido para todas as palavras em inglês dessa blade